### ISSUE TITLE
No way to forcibly invalidate cache

### ISSUE THREAD

**Issue by AlbinoDrought** [AlbinoDrought/cachios#8]
Usecase:

I have a list of resources that a user will frequently view - I want to cache these. However, after performing a destructive action, I want to get the current, uncached list of resources. 

There is no way to currently do this without manually invalidating the cache.

Possible solution:

~Magic config key, something like~ implemented in #45 

```js
cachios.get('items', {
   forced: true
});
```

which could be checked at https://github.com/AlbinoDrought/cachios/blob/master/src/cachios.js#L53

```js
  if (cachedValue === undefined || config.forced) {
    promise = this.axiosInstance.request(config).then((resp) => {
      this.setCachedValue(cacheKey, this.getResponseCopy(resp), ttl);
      return resp;
    });
  } else {
    promise = Promise.resolve(cachedValue);
  }
```

**Comment by Aidurber** [comment]
What determines a destructive action? Is it DELETE, POST, PUT, and PATCH verbs?

**Comment by liubko** [comment]
Have a similar situation, want to cache get requests, and I know that after certain POST requests some of the cached requests will be invalid. Would like to see an ability to clean some certain parts in the cache.

**Comment by AlbinoDrought** [comment]
A way to forcibly invalidate cache on a request-by-request basis has been implemented in #45. 

At the time of this comment, the title of this issue is `No way to forcibly invalidate cache`. There is now a way to forcibly invalidate cache, so I am going to close this issue.

If needed, please leave a comment and it can be renamed and reopened.
